src/renderer/components/outputs/LargeImageOutput.tsx

```diff
@@ -0,0 +1,176 @@
+            if (value) {
+                setStale(false);
+            }
+        }, [value]);
+
+        useEffect(() => {
+            if (value && lastInputDataUpdatedId !== id) {
+                setStale(true);
+            }
+        }, [inputDataChanges]);
+
+        useEffect(() => {
+            // Run this only if this is a "starting" node
+            if (!schema.inputs.some((i) => i.hasHandle)) {
+                if (value) {
+                    setManualOutputType(
+                        id,
+                        outputId,
+                        new NamedExpression('Image', [
+                            new NamedExpressionField('width', new NumericLiteralType(value.width)),
+                            new NamedExpressionField(
+                                'height',
+                                new NumericLiteralType(value.height)
+                            ),
+                            new NamedExpressionField(
+                                'channels',
+                                new NumericLiteralType(value.channels)
+                            ),
+                        ])
+                    );
+                } else {
+                    setManualOutputType(id, outputId, undefined);
+                }
+            }
+        }, [id, schemaId, value]);
+
+        const imgBgColor = useColorModeValue('gray.400', 'gray.750');
+        const fontColor = useColorModeValue('gray.700', 'gray.400');
+
+        return (
+            <Center
+                h="full"
+                minH="2rem"
+                overflow="hidden"
+                verticalAlign="middle"
+                w="full"
+            >
+                <Center
+                    h="200px"
+                    // overflow="hidden"
+                    w="200px"
+                >
+                    <Box
+                        display={stale ? 'block' : 'none'}
+                        h="200px"
+                        marginRight="-200px"
+                        w="200px"
+                        zIndex="99"
+                    >
+                        <HStack
+                            alignContent="center"
+                            alignItems="center"
+                            bgColor={imgBgColor}
+                            borderRadius="lg"
+                            margin={1}
+                            opacity={0.75}
+                            px={2}
+                            py={1}
+                            spacing={0.5}
+                            verticalAlign="middle"
+                            w="4rem"
+                            zIndex="99"
+                        >
+                            <WarningIcon
+                                boxSize={2}
+                                color={fontColor}
+                            />
+                            <Text
+                                color={fontColor}
+                                fontSize="xx-small"
```

Comment:
For how important this information is, the font is mighty small.
![image](https://user-images.githubusercontent.com/20878432/180607857-2d805241-1025-4db8-931d-575d65151aab.png)

Reply:
It's also misaligned for you for some reason